job-dsls/jobs/new_pr_jobs.groovy

```diff
@@ -201,7 +201,7 @@ for (repoConfig in REPO_CONFIGS) {
                 orgslist("${ghOrgUnit}")
                 whitelist("")
                 cron("")
-                triggerPhrase(".*[j|J]enkins,?.*(retest|test) this please.*")
+                triggerPhrase(".*[j|J]enkins,?.*(retest|test) this.*")
```

Comment:
this is ok to remove please word even to be polite you can still include it no matter what phrase reges is ;)